### PR TITLE
Add migration for skills and projects

### PR DESCRIPTION
**Pull request by aaronreba**
This PR also includes an update to .gitignore for vim. Please let me know if this should be excluded from this PR.

Addresses https://trello.com/c/MpQNkJpO/175-voma-update-schema

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -10,11 +10,55 @@
...
+  create_table "volunteers", id: :serial, force: :cascade do |t|
+    t.string "name"
+    t.string "email"
+    t.string "slackUserId"
+    t.string "pronouns"
+    t.string "primarySkills"
+    t.string "secondarySkills"
+    t.string "employer"
+    t.string "student"
+    t.string "assignedProjects"
```
Comment (dkparker) [inline db/schema.rb]: This seems redundant to me with the volunteer_projects table, but maybe I'm misinterpreting the purpose.

Comment (aaronreba) [reply db/schema.rb]: Yeah, along with the skills columns on this table. Will remove.

**Comment by dkparker** [comment]
Also sorta looks like the ideal workflow in the miro board may have interest in tracking date of onboarding meeting attendance and 1-on-1. Or at least, the existence or lack of these. Maybe nullable date fields for these would be appropriate @tajaran ?

**Comment by tajaran** [comment]
> Also sorta looks like the ideal workflow in the miro board may have interest in tracking date of onboarding meeting attendance and 1-on-1. Or at least, the existence or lack of these. Maybe nullable date fields for these would be appropriate @tajaran ?

Yes it would be ideal for us to track any meeting occurrence dates. When you say nullable date fields, what do you mean by that? Sorry, not sure what that looks like from a design  perspective.

**Comment by dkparker** [comment]
Oh, just that it would have the capability of indicating that there has not been that type of meeting yet. Ok cool, thanks for the confirmation!

**Comment by tajaran** [comment]
> Oh, just that it would have the capability of indicating that there has not been that type of meeting yet. Ok cool, thanks for the confirmation!

Oh gotcha! Thanks for clarifying, You're right in adding nullable date fields for meetings that haven't happened yet would work.

**Comment by aaronreba** [comment]
Good suggestion on the new date fields. Added in latest changes.

File: spec/controllers/skills_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe "API V1 Skills", type: :request do
```
Comment (Ryan-Koch) [inline spec/controllers/skills_controller_spec.rb:3]: Thanks for adding rspec tests!

**Review by Ryan-Koch — APPROVED** [review]

**Pull request merged**